task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +115,17 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return function TryFunc(){
+        for (; attempts > 0;){
+            try {
+                return func();
+            }
+            catch (e){
+                attempts-=1;
```

Comment:
Do not modify funciton arguments if not required. 99% of the time you don't need to do it.

Reply:
Fixed